server/routerlicious/kubernetes/nginx/config/nginx.tmpl

```diff
@@ -198,7 +198,7 @@ http {
     server_name_in_redirect off;
     port_in_redirect        off;
 
-    ssl_protocols {{ $cfg.SSLProtocols }};
```

Comment:
By the way @yunho-microsoft , where does .Cfg come from? Do we have to pass it manually when installing the nginx chart? We talked about the .tmpl file the other day but I couldn't figure out where the .Cfg is coming from.

Reply:
I haven't figured it out. We don't pass it manually. I think .Cfg means the default setting for Nginx but I don't find any documents about it. For the syntax of the tmpl file, this is the doc: https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/custom-template/

From the doc, there is a link to the template file syntax: https://pkg.go.dev/text/template
I didn't spend much time on it yet.